tortoise/ninja_tortoise.go

```diff
@@ -586,7 +586,7 @@ func (ni *ninjaTortoise) GetGoodPattern(layer types.LayerID) (uint32, error) {
 		return uint32(getId(v)), nil
 	}
 
-	if layer >= ni.pBase.LayerID {
+	if layer > ni.pBase.LayerID {
```

Comment:
why is this a part of this pr?

Reply:
Because the automation test I adjusted discovered this bug